src/NetMQ.Tests/ProxyTests.cs

```diff
@@ -1,29 +1,13 @@
 ﻿using System;
 using System.Threading.Tasks;
 using NetMQ.Sockets;
-using NUnit.Framework;
+using Xunit;
 
 namespace NetMQ.Tests
 {
-    [TestFixture]
-    public class ProxyTests
+    public class ProxyTests : IClassFixture<NetMQConfigFixture>
     {
-        [OneTimeSetUp]
-        public void SetUp()
```

Comment:
I think this is the reason the tests fail.
Does xUnit has an alternative for this?